src/content/concepts/mode.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Mode
+sort: 4
+contributors:
+  - EugeneHlushko
+---
+
+Providing the `mode` configuration option tells webpack to use its built-in optimizations accordingly.
+
+`string`
+
+## Usage
+
+Just provide the `mode` option in the config:
+
+```javascript
+module.exports = {
+  mode: 'production'
+};
+```
+
+ 
+or pass it as a cli argument:
+
+```bash
+webpack --mode=production
```

Comment:
`--mode production`

Reply:
Can we also mention the precedence of `mode` if it's present in both config and via cli?

Reply:
`--mode=production` is valid too

Reply:
`--mode production` looks cleaner to me. Anyways, both are fine. :1